lib/algora/terminate.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Algora.Terminate do
+  use GenServer
+
+  import Ecto.Query, warn: false
+
+  @terminate_interval :timer.se(10)
```

Comment:
Typo here, let's also increase the interval to 1 hour

Reply:
Whoops, done! New timer is for 1 hour